packages/cadl-ranch-specs/http/payload/multipart/main.tsp

```diff
@@ -65,6 +65,11 @@ Content-Type: text/plain
 
 123
 --abcde12345
+Content-Disposition: form-data; name="temperature"
+Content-Type: text/plain
```

Comment:
Does that mean following settings?
- for primitive types the default content type is `text/plain`
- for objects(object, array, record etc) the default content type is `application/json`

I am asking because for `string` if it is json content, it would be `"raw string"`; if it is text content, then would be `raw string`.

Reply:
I understand what you mean. cadl-ranch uses `multer` to parse and it has limitation: different with `files`, `data` part lose content-type info so please pass it as plain string instead of json string. 
![image](https://github.com/Azure/cadl-ranch/assets/70930885/461abb3f-f681-41bc-9e4f-0c4bc570d766)
 In the future, maybe I can find another package to get the info then the case can handle it with different content types.

Reply:
I think it still `text/plain`. Admit this be the same problem of ambiguous on content-type.

AFAIK, for current openai impl, the string and float would result to a text without quote.
```java
        if (audioTranscriptionOptions.getPrompt() != null) {
            fields.add(new MultipartField("prompt",
                audioTranscriptionOptions.getPrompt()));
        }
        if (audioTranscriptionOptions.getTemperature() != null) {
            fields.add(new MultipartField("temperature",
                String.valueOf(audioTranscriptionOptions.getTemperature())));
        }
```
(float temperature is converted to string)

Reply:
@MaryGao, @weidongxu-microsoft I update this case with `HttpPart` to declare contentType with `text/plain` so there shall be no ambiguity now.